kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/utils/MetadataUtils.java

```diff
@@ -98,6 +98,10 @@ private static void createKafkaMetadataIfMissing(String tenant,
                                                      KopTopic kopTopic,
                                                      int partitionNum)
         throws PulsarAdminException {
+        if (!conf.isKafkaManageSystemNamespaces()) {
```

Comment:
The config name is better to be `skipCreatingKafkaMetadata` (and the default value is false) according to the method name.

Reply:
Initially I thought about it but
it is not only about "creating" but it is also about setting retention and other parameters

we don't want that parameters set by the user are overridden by the PH.

think about an environment where there are tons of independent tenants, and you want total independence of the configurations (and also you want that some system control plane is in change of managing system namespaces)

Reply:
Can we call the config name `kopAutoManageMetadataEnable`(the default value is true)? Do you have a better idea?

Reply:
Ping @eolivelli. Is there any update?  I think it's only a matter of config name.

Reply:
Make sense.